packages/sources/src/components/tabs/tabs.args.ts

```diff
@@ -0,0 +1,21 @@
+import { ArgTypes } from '../../stories-helpers';
+
+import { Tabs, TabItem } from './tabs.models';
+
+export interface TabsArgs {
+  tabs: TabItem[];
```

Comment:
Ik denk dat we de tabs/items niet via Args moeten lopen, maar wat dan wel weet ik nog niet.